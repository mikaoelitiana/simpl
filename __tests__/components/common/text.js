import React from "react";
import { mount } from "enzyme";
import Text from "./../../../src/components/text";

it("renders as snapshot", () => {
  const rendered = mount(<Text>Hello World</Text>);
  expect(rendered).toMatchSnapshot();
});