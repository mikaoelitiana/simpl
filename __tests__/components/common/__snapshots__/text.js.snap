// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as snapshot 1`] = `
<CustomText>
  <Component
    style={
      Array [
        Object {
          "fontFamily": "Ubuntu-Regular",
        },
        null,
        undefined,
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontFamily": "Ubuntu-Regular",
          },
          null,
          undefined,
        ]
      }
    >
      Hello World
    </Text>
  </Component>
</CustomText>
`;
